pages/common/composer.md

```diff
@@ -0,0 +1,31 @@
+# composer
+
+> Dependency Manager for PHP.
+
+- Interactively create a composer.json file defining the project's dependencies:
+
+`composer init`
+
+- Create a new project from a given version of an existing package:
+
+`composer create-project {{project_name}} {{/path/to/my/project}} {{version}}`
+
+- Install the packages specified in composer.json file in the current directory:
```

Comment:
I was under the impression that this would always be done in a *specific* subdirectory of the project's root (as indicated by the location of the composer.json file). This description makes it seem like whatever directory one is within the project, there's where the packages will be installed. Considering the contents of the [intro page](https://getcomposer.org/doc/00-intro.md) (emphasis mine):

> [composer manages dependencies] on a per-project basis, installing them in **a directory** inside your project"

...should this description be reworded? Or am I misreading something?